Compilation error - Fedora  42, x86_64

Using git source.

Getting to the tools build section and failing as such :

```
In file included from code/tools/lcc/src/alloc.c:1:
code/tools/lcc/src/c.h:648:1: error: ‘constexpr’ used with ‘extern’
  648 | extern Tree constexpr(int);
      | ^~~~~~
code/tools/lcc/src/c.h:648:23: error: expected identifier or ‘(’ before ‘int’
  648 | extern Tree constexpr(int);
      |                       ^~~
make[2]: *** [Makefile:1557: build/release-linux-x86_64/tools/rcc/alloc.o] Error 1
```

Any ideas what I am doing wrong?

TIA,
Andrew.

--- a/code/tools/lcc/src/c.h
+++ b/code/tools/lcc/src/c.h
@@ -645,7 +645,7 @@ extern int process(char *);
 extern int findfunc(char *, char *);
 extern int findcount(char *, int, int);
 
-extern Tree constexpr(int);
+extern Tree const_expr(int);
 extern int intexpr(int, int);
 extern Tree simplify(int, Type, Tree, Tree);
 extern int ispow2(unsigned long u);

--- a/code/tools/lcc/src/simp.c
+++ b/code/tools/lcc/src/simp.c
@@ -182,7 +182,7 @@ static int subi(long x, long y, long min, long max, int needconst) {
 static int subd(double x, double y, double min, double max, int needconst) {
 	return addd(x, -y, min, max, needconst);
 }
-Tree constexpr(int tok) {
+Tree const_expr(int tok) {
 	Tree p;
 
 	needconst++;
@@ -192,7 +192,7 @@ Tree constexpr(int tok) {
 }
 
 int intexpr(int tok, int n) {
-	Tree p = constexpr(tok);
+	Tree p = const_expr(tok);
 
 	needconst++;
 	if (p->op == CNST+I || p->op == CNST+U)

--- a/code/tools/lcc/src/stmt.c
+++ b/code/tools/lcc/src/stmt.c
@@ -119,7 +119,7 @@ void statement(int loop, Swtch swp, int lev) {
 		       		static char stop[] = { IF, ID, 0 };
 		       		Tree p;
 		       		t = gettok();
-		       		p = constexpr(0);
+		       		p = const_expr(0);
 		       		if (generic(p->op) == CNST && isint(p->type)) {
 		       			if (swp) {
 		       				needconst++;

